runtime/error.c

```diff
@@ -17,6 +17,13 @@
 #define EXCEEDS_MAX_PC_COUNT -3
 #define OUT_OF_MEMORY -4
 
+#define NON_PUBLIC_SYMBOL_PREFIX "_B_"
+#define NON_PUBLIC_SYMBOL_PREFIX_LENGTH strlen(NON_PUBLIC_SYMBOL_PREFIX)
```

Comment:
Hopefully clang optimize this to a consent. @KavinduZoysa please verify.

Reply:
Yes, it is optimized to a constant.